examples/counter_out/cont_gen_dig_pulse_train.py

```diff
@@ -0,0 +1,21 @@
+"""Example for continuously generating digital pulse train.
+
+This example demonstrates how to generate a continuous digital
+pulse train from a Counter Output Channel. The Frequency, Duty
+Cycle, and Idle State are all configurable.
+"""
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, Level
+
+with nidaqmx.Task() as task:
+    channel = task.co_channels.add_co_pulse_chan_freq(
+        "Dev1/ctr1", idle_state=Level.LOW, initial_delay=0.0, freq=100.0, duty_cycle=0.5
```

Comment:
The C examples default to 1 Hz

`DAQmxErrChk (DAQmxCreateCOPulseChanFreq(taskHandle,"Dev1/ctr0","",DAQmx_Val_Hz,DAQmx_Val_Low,0.0,1.00,0.50));`

Again, a lower rate is probably more visible without an oscilloscope or ctr/timer.

Reply:
freq is set to 1.0 for all freq examples.